server/server.py

```diff
@@ -2,12 +2,15 @@
 	import socketserver
 	import time
 	import sqlite3
+	import argparse
+	from ipaddress import ip_address
+	from sys import argv
```

Comment:
Is _from sys import argv_ needed? It doesn't exist in client.py, and I don't see any real differences that would need it to be imported.